stable/datadog/templates/agent-clusterchecks-deployment.yaml

```diff
@@ -10,6 +10,15 @@ metadata:
     heritage: {{ .Release.Service | quote }}
 spec:
   replicas: {{ .Values.clusterchecksDeployment.replicas }}
+  strategy:
+{{- if .Values.clusterchecksDeployment.strategy }}
+{{ toYaml .Values.clusterchecksDeployment.strategy | indent 4 }}
+{{- else }}
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
```

Comment:
since usually we have only few instances, do we want to have `maxUnavailable: 1` or `maxUnavailable: 0`

Reply:
made it 0 👌